<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>上海海事大学统一认证平台验证码OCR Demo QT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[等待OCR]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Author">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Author:Haomin Kong</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ImageView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[当前未加载图片]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DownloadUrl">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>下载图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OpenLocal">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>打开本地图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ocr">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>OCR识别</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>模型目录：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CheckpointPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结果：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_UseGpu">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>77</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use GPU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ReleaseModel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>释放模型</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>程序</string>
    </property>
    <addaction name="actionReleaseModel"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionDownload_From_URL"/>
    <addaction name="actionOpenLocal"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionDoOCR"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionReleaseModel">
   <property name="text">
    <string>释放模型</string>
   </property>
  </action>
  <action name="actionDownload_From_URL">
   <property name="text">
    <string>下载图片</string>
   </property>
  </action>
  <action name="actionOpenLocal">
   <property name="text">
    <string>打开本地图片</string>
   </property>
   <property name="iconText">
    <string>OpenLocal</string>
   </property>
   <property name="toolTip">
    <string>OpenLocal</string>
   </property>
  </action>
  <action name="actionDoOCR">
   <property name="text">
    <string>执行OCR</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
